src/components/ContentSidebar/ActivityFeed/version/README.md

```diff
@@ -0,0 +1,17 @@
+## Comment
+
+Comment primitive
+
+### Usage
```

Comment:
is this file needed?

Reply:
Gar, forgot to remove READMEs. Doing it now